docs/src/main/paradox/consumer.md

```diff
@@ -115,25 +107,22 @@ Java
 : @@ snip [dummy](../../test/java/sample/javadsl/ConsumerExample.java) { #groupedWithin }
 
 
-If you commit the offset before processing the message you get "at-most once delivery" semantics, and for that there is a `Consumer.atMostOnceSource`. However, `atMostOnceSource` commits the offset for each message and that is rather slow, batching of commits is recommended.
+If you commit the offset before processing the message you get "**at-most** once" delivery semantics, this is provided by `Consumer.atMostOnceSource`. However, `atMostOnceSource` commits the offset for each message and that is rather slow, batching of commits is recommended.
```

Comment:
Here only the "at-most" part is highlighted. A bit below whole "at-least-once" is highlighted and in some other places nothing is highlighted in "at-least once". We should keep it consistent regarding the "hyphenization" and I think no bold is fine:

* at-least-once
* at-most-once